src/modules/job-manager/queue.c

```diff
@@ -46,6 +46,7 @@ struct jobq {
     bool checkpoint_start;  // may be different that actual start due
                             // to nocheckpoint flag
```

Comment:
Commit title:   totally minor but I think we've been using "topic/subtopic:" where a subtopic is needed.  "job-manager: queue:" doesn't  fit that pattern.

Reply:
Hm, I think this seems to be personal preference:

```
 grondo@pi3:~/git/flux-core.git$ git log | grep -c job-manager:.*: 
47
 grondo@pi3:~/git/flux-core.git$ git log | grep -c job-manager/ 
57
```

BTW, I don't necessarily like `topic/subtopic` because when you search for `topic: ` thinking you'll find all matches for `topic` you are mistaken :-)